site-src/concepts/versioning.md

```diff
@@ -2,59 +2,87 @@
 
 ## Overview
 Each new release of Gateway API is defined with a "bundle version" that
-represents the Git tag of a release, such as v0.4.0. This contains the
+represents the Git tag of a release, such as v0.8.0. This contains the
 following:
 
 * API Types (Go bindings for the resources)
 * CRDs (Kubernetes definitions of the resources)
-* Validating Webhook
-
-### API Versions (e.g. v1alpha2, v1beta1)
-Within Gateway API, API versions are primarily used to indicate the stability of
-a resource. For example, if a resource has not yet graduated to beta, it is
-still possible that it could either be removed from the API or changed in
-backwards incompatible ways. For more information on API versions, refer to the
-[full Kubernetes API versioning
-documentation](https://kubernetes.io/docs/reference/using-api/#api-versioning).
-
-![Lifecycle of New Gateway API Resources](/images/lifecycle-new-resources.png)
-<!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
-
-### Release Channels (e.g. Experimental, Standard)
-Gateway API provides 2 release channels: an Experimental one and a Standard one.
+* Validating Webhook (Will be deprecated in v1.0 release)
+
+### Release Channels
+Release channels are used to indicate feature stability within Gateway API. All
+new features and resources start in the Experimental release channel. From that
+point, these may graduate to the Standard release channel or be dropped from the
+API entirely.
+
+The following diagram provides a high level overview of the lifecycle of a
+feature or resource proposed by a new [GEP](/geps/overview/) (enhancement
+proposal) in Gateway API:
+
+```mermaid
+flowchart TD
+    0([Implementable GEP]) --> A
+    A>Experimental Channel] --> B([Widely used and working well?])
+    B -->|Yes| C>Standard Channel]
+    B -->|No| D([Could Changes Help?])
+    D -->|Yes| E([Adjust and try again])
+    D -->|No| F>Remove From API]
+    E -->A
+
+style A fill:#eeb
+style C fill:#beb
+style F fill:#ebb
+```
 
 The Standard release channel includes:
 
-* Resources that have graduated to beta
-* All fields that have graduated to standard and are no longer considered
-  experimental
+* Resources that have graduated to Beta or GA API Versions
```

Comment:
Is it worth adding something like "Note that no new CRDs will get Beta versions" or the like?

Reply:
Updated, thanks!